packages/blade/src/components/Popover/_decisions/decisions.md

```diff
@@ -0,0 +1,275 @@
+| **Content**             	| Heading<br>Body Text (Normal + Bold)            	| Heading<br>Rich Body Text<br>Links<br>Buttons<br>Assets/ Media 	| Heading<br>Rich Body Text<br>Links<br>Buttons<br>Assets/ Media 	|
+| **Header**              	| Plain Text                                       	| Rich Text<br>Leading Icon<br>Leading Asset                     	| Rich Text<br>Leading Icon<br>Leading Asset                     	|
+| **Footer**              	| No                                               	| Slot<br>Action Buttons(1 or 2)                                	| Action Buttons(1 or 2)<br>Nav Buttons (2)                     	|
+| **Dismiss Button**      	| No                                               	| Yes                                                            	| Yes                                                            	|
+| **Background Overlay**  	| No                                               	| No                                                             	| Yes                                                            	|
+
+
+## Library
+
+We will be using [FloatingUI](https://floating-ui.com/) to position the popover & handle the basic popover logic. Floating UI supports both Web & ReactNative.
+
+## Motion
+
+You can check the [popover motion here](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=2875-43270&mode=design&t=mTbpUCQOr7kHk6UT-0).
+
+
+https://github.com/razorpay/blade/assets/35374649/f774e28f-bbb7-4f44-a41d-71e9b873ec0c
+
+
+And give relavant feedback for [motion in this thread](https://razorpay.slack.com/archives/C0274H7QRC1/p1691645251106119).
+
+## Accessibility
+
+- Popover will follow the [modaldialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) WAI-ARIA pattern.
+- This can be achived with FloatingUI's [`modal` prop](https://floating-ui.com/docs/popover#modal)
+
+## Open Question
+
+How should the footer API look like? 
+
+- Should we keep footer flexible and accept React.ReactNode? 
+- Or should we follow similar API like Alert? 
+
+**Flexible API:**
+
+```jsx
+<Popover
+  content={<Text>Any content</Text>}
+  footerContent={
+    <Box display="flex">
+      <Button>Primary me</Button>
+      <Button variant="secondary">Secondary me</Button>
+    </Box>
+  }
+>
+  <IconButton icon={InfoIcon} />
+</Popover>
+```
+
+**Contrained API:** 
+
+```jsx
+<Popover
+  content={<Text>Any content</Text>}
+  footerContent={<Text>Slotted content of Footer</Text>}
+  footerActions={{
+    primary: {
+      text: "Primary me",
+      onClick: () => {},
+    },
+    secondary: {
+      text: "Secondary me",
+      onClick: () => {},
+    },
+  }}
+>
+  <IconButton icon={InfoIcon} />
+</Popover>
+```
+
+**Conclusion:** We should go ahead with Flexible API because even if we go with constrained API, users will anyway still be able to pass React.ReactNode as `footerContent` and could just skip the `footerActions` altogether.
```

Comment:
```suggestion
**Conclusion:** We will go ahead with Flexible API as this will mostly be controlled by design and design has the slots and permutations and combinations in place of what is allowed and what is disallowed.
```